src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Color;
+import core.basesyntax.Drawable;
+
+public abstract class Figure implements Drawable {
+    protected Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public void draw() {
+        System.out.println("Figure : ");
```

Comment:
it isn't an informative realization. Let’s create an informative message or better remove it

Reply:
made it abstract